Sources/Jobs/Scheduler/Schedule.swift

```diff
@@ -12,6 +12,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+import Collections
 #if os(Linux)
```

Comment:
Is this flag still needed?

Reply:
Yes Calendar still hasn't been marked as Sendable on Linux, it might not be needed on Swift 6